LICs.c

```diff
@@ -208,6 +208,40 @@ boolean check_lic_7(void){
     return false;
 }
 
+//LIC 8
+boolean check_lic_8(void){
+    if(PARAMETERS.APTS < 1 || PARAMETERS.BPTS < 1 ||
+    PARAMETERS.APTS + PARAMETERS.BPTS > NUMPOINTS - 3)
+        return false;
+
+    if(NUMPOINTS < 5)
+        return false;
+
+    for(int i = 0; i + PARAMETERS.APTS + PARAMETERS.BPTS < NUMPOINTS - 3; i++) {
+        int a = i + PARAMETERS.APTS;
+        int b = a + PARAMETERS.BPTS;
+
+        double x[3] = {*(X+i), *(X+a), *(X+b)};
+        double y[3] = {*(Y+i), *(Y+a), *(Y+b)};
+
+        if (x[0] == x[1] && y[0] == y[1] ||
+        x[0] == x[2] && y[0] == y[2] ||
+        x[1] == x[2] && y[1] == y[2])
+            continue;
```

Comment:
I don't understand why having the same point twice would be an issue. Is there something I am missing?

Reply:
No, I mixed it up with a previous LIC. I'll fix that!